src/encoded/audit/biosample.py

```diff
@@ -346,18 +346,18 @@ def audit_biosample_part_of_consistency(value, system):
 
         ontology = system['registry']['ontology']
         if (term_id in ontology) and (part_of_term_id in ontology):
-            ontology_child = ontology[term_id]
+            # ontology_child = ontology[term_id]
             ontology_parent = ontology[part_of_term_id]
             if 'name' in ontology_parent and ontology_parent['name'] == 'multi-cellular organism':
                 return
-
+            '''
             if 'organs' in ontology_child and 'organs' in ontology_parent:
                 child_organs = ontology_child['organs']
```

Comment:
I don't think you should comment out this code unless you plan on bringing it back very soon.